### PR TITLE
minor revisions to open science lessons for clarity

### PR DESCRIPTION
**Pull request by dlebauer**
migration of pull request swcarpentry/bc#919

@chinchliff could you please review and provide feedback on these changes?

### CODE REVIEW

**Comment by gvwilson** [comment]
Is this one ready to merge?

File: 04-open.md [hunk 1]
```diff
@@ -208,11 +209,12 @@ most give users a choice:
...
 it will be hosted for free,
 but if they want privacy,
-they may have to pay.
-Sharing might seem like the only valid choice for science,
-but many institutions may not allow researchers to do this,
-either because they want to protect future patent applications
-or simply because what's new is often also frightening.
+there may be limits on the number of private repositories or collaborators.
+Access to private repositories can be purchased through 'premium plans', although GitHub and Bitbucket offer free private repositories for academic and / or open-source projects.
+Sharing might seem an ideal approach for scientific code,
```
Comment (dlebauer) [inline 04-open.md]: Good point now it's 'sharing is the ideal for science...'

**Comment by dlebauer** [comment]
@jiffyclub I think I've addressed your points, mostly copy pasted. Please feel free to make further revisions directly, or merge

**Comment by jiffyclub** [comment]
Looks good, thanks!

**Pull request merged**